### ISSUE TITLE
UI Hud disables when installed addon DIALOGUE UI

### ISSUE THREAD

**Issue by Xpatro** [p3lim-wow/InteractiveWormholes#68]
**0.2 sec interaction and CLOSE when installed Dialogue UI + Interactive Wormhole**

https://www.curseforge.com/wow/addons/dialogueui

**as mentioned above**

**If a player having both addons, and option when interacted with NPC (bottom right corner) "map-locations"**

**Comment by Xpatro** [comment]
I would suggest alternative interact button (CTRL) if so to force-map Overlay "teleport" view to see

**Comment by Xpatro** [comment]
Any update??

**Comment by filliph** [comment]
@p3lim When you use Dialogue UI, it replaces the normal dialogue box with a highly stylised one: 

![wowscrnshot_032124_205644](https://github.com/p3lim-wow/InteractiveWormholes/assets/9424286/829afae2-86fa-49b7-804d-b628381294ac)

As you can see, it also fades out the normal game UI.

However, with this add-on installed, since both your add-on and it tries to override the Gossip UI, it instead flashes to the world map and flashes the Dialogue UI pop-up, before closing both.

In other words, it becomes impossible to interact with either of your two add-ons.

It would be greatly appreciated if you could try to figure out a way to make both of them work 😄

**Comment by p3lim** [comment]
After looking at Dialogue UI there doesn't seem like there is any way to integrate with it, e.g. choose which ones gets to "control" the gossip.

If you can convince the author of the addon to cooperate with me I'd be more than happy to make it work.

**Comment by p3lim** [comment]
Compatibility added in DialogueUI 0.3.6: https://github.com/Peterodox/YUI-Dialogue/issues/26
